twitter-connector-product/pom.xml

```diff
@@ -26,6 +25,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks>
+                <unzip dest="">
```

Comment:
use dest="${basedir}" in order to reference the current directory with an absolute path.
otherwise you may have strange effects if you run this build from another location unexpected location
.eg. via mvn clean -f dev/my-connector/pom.xml